site/src/nav/DownloadButton.css

```diff
@@ -26,3 +26,37 @@
     display: none;
   }
 }
+
+div.download-icon {
+  width:20px;
```

Comment:
As mentioned below, let's reformat with prettier to keep things consistent. There's a missing space between width and the pixels.